Adds docs for new CCEI module changes (#26)
msupply-foundation/msupply_docs: content/docs/catalogue/assets.md

+++
title = "Assets"
description = "Viewing assets."
date = 2022-03-19T18:20:00+00:00
updated = 2022-03-19T18:20:00+00:00
draft = false
weight = 52
sort_by = "weight"
template = "docs/page.html"

[extra]

> lead = "Managing your assets"

Reviewer: maybe..?
```suggestion
lead = "Managing your catalogue of asset items"
```

toc = true
top = false
+++

> Assets in Open mSupply is a digital register to manage the creation and maintenance of assets. It allows health departments to keep track of, manage and maintain their assets.

Reviewer: ```suggestion
Assets in Open mSupply is a digital register to manage the creation and maintenance of assets. 

The asset catalogue provides a list of base asset items, and is pre-populated from the [WHO PQS Catalogue](https://apps.who.int/immunization_standards/vaccine_quality/pqs_catalogue/) (sections E003 & E004). You can then create an asset for each piece of equipment at a site using the [Equipment](/docs/coldchain/equipment/) section, with these assets based on the items in this catalogue. Doing so will give you base information, such as the manufacturer & model of an asset without having to enter that manually.
```

Author: I've changed this to ```The asset catalogue provides a list of base asset items, and is pre-populated from the [WHO PQS Catalogue](https://apps.who.int/immunization_standards/vaccine_quality/pqs_catalogue/) (sections E003 & E004). You can then create an asset based on the items in this catalogue using the [Equipment](/docs/coldchain/equipment/) section. Doing so will give you base information, such as the manufacturer & model of an asset without having to enter that manually.```

as it felt a bit easier to digest for me, but let me know if you think it doesn't offer enough information

From the **Assets** menu, you can view all of the assets currently in your store.

### Viewing Asset List

In the navigation panel, tap on `Catalogue` > `Assets` to show the asset list

![Assets nav](/docs/catalogue/images/assets.png)

Here you can see all the assets which are available in your store.

The asset list is divided into 6 columns:

| Column            | Description                                                |
| :---------------- | :--------------------------------------------------------- |
| **Sub catalogue** | The catalogue this asset belongs to                        |
| **Code**          | The code of the catalogue item which this asset belongs to |
| **Type**          | The type of asset                                          |
| **Manufacturer**  | The manufacturer of your asset                             |
| **Model**         | The model number of the asset                              |
| **Class**         | The class of the asset. ie: `Cold chain equipment`         |

> | **Category**      | The subcategory of the asset                               |

Reviewer: ```suggestion
| **Category**      | The subcategory of the asset e.g. section E003 in the PQS catalogue which is `Refrigerators and freezers`                               |
```

![Assets page](/docs/catalogue/images/assets_page.png)

The list can display a fixed number of assets per page. On the bottom left corner, you can see how many assets are currently displayed on your screen.

![Page](/docs/distribution/images/os_list_showing.png)

If you have more assets than the current limit, you can navigate to the other pages by tapping on the page number or using the right of left arrows (bottom right corner).

![Page](/docs/distribution/images/os_list_pagenumbers.png)

You can also select a different number of rows to show per page using the option at the bottom right of the page.

![Rows per page](/docs/introduction/images/rows-per-page-select.png)

#### Filter Assets

To add a filter to the page, choose the required filter from the drop down. Multiple filters can be combined.

<p align="center">
    <img src="/docs/catalogue/images/assets_filter.png" width="425" height="250">
</p>

#### Delete Assets

Assets can be selected and deleted using the toolbar action:

<p align="center">
    <img src="/docs/catalogue/images/assets_delete.png" width="425" height="380">
</p>

### Importing And Exporting

#### Import

> Assets can be imported from a csv file using the `Import` button

Reviewer: ```suggestion
Assets can be imported from a comma-separated-file (csv) file using the `Import` button
```

![Assets nav](/docs/catalogue/images/assets_import.png)

This will open an import modal.

<p align="center">
    <img src="/docs/catalogue/images/assets_import_modal.png" width="750" height="420">
</p>

> An example template comma-separated-value (csv) is available for download here:

Reviewer: ```suggestion
An example template (in csv format) is available for download here:
```

<p align="center">
    <img src="/docs/catalogue/images/asset_import_template.png" width="750" height="420">
</p>

Data will need to be converted into the format of the csv template provided in order for Open mSupply to be able to process and upload this data.

A csv file can be uploaded once it has been created in the example format.

#### Export

> Asset can be exported as a csv using the `Export` button

Reviewer: ```suggestion
A list of assets can be exported as a csv file using the `Export` button
```

![Assets nav](/docs/catalogue/images/asset_export_button.png)

### Managing status log reasons

> Status log reasons can be managed using this button.

Reviewer: This line just confused me a little and I think it can be removed. Ideally the button would be shown directly under the line, but you into the button below too, and that's next to the image.

The asset list is divided into 6 columns:

| Column            | Description                                                |
| :---------------- | :--------------------------------------------------------- |
| **Sub catalogue** | The catalogue this asset belongs to                        |
| **Code**          | The code of the catalogue item which this asset belongs to |
| **Type**          | The type of asset                                          |
| **Manufacturer**  | The manufacturer of your asset                             |
| **Model**         | The model number of the asset                              |
| **Class**         | The class of asset. ie: `Cold chain equipment`             |
| **Category**      | The subcategory of the asset                               |

![Assets page](/docs/catalogue/images/assets_page.png)

To add a filter to the page, choose the required filter from the drop down. Multiple filters can be combined.

![Assets filter](/docs/catalogue/images/assets_filter.png)

Assets can be selected and deleted using the toolbar action:

![Assets nav](/docs/catalogue/images/assets_delete.png)

### Importing And Exporting

#### Import

> Assets can be imported from a csv file using the 'Import' button

Reviewer 2: ```suggestion
Assets can be imported from a csv file using the `Import` button
```

![Assets nav](/docs/catalogue/images/assets_import.png)

This will open an import modal.

![Assets nav](/docs/catalogue/images/assets_import_modal.png)

An example template comma-separated-value (csv) is available for download here:

![Assets nav](/docs/catalogue/images/asset_import_template.png)

Data will need to be converted into the format of the csv template provided in order for Open mSupply to be able to process and upload this data.

A csv file can be uploaded once it has been created in the example format.

#### Export

> Asset can be exported as a csv using the 'Export' button

Reviewer 2: ```suggestion
Asset can be exported as a csv using the `Export` button
```
